Include/pystate.h

```diff
@@ -28,7 +28,6 @@ typedef struct _is {
     struct _is *next;
     struct _ts *tstate_head;
 
-    PyObject *modules;
```

Comment:
I don't think we can remove a field in the middle of the struct, as that would break the stable ABI (perhaps @zooba  can confirm). Instead probably replace it with `void *_unused` or something.